cdk/bin/pipeline.ts

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
```

Comment:
Why not add this stack as part of the root api.ts? I think naming is a bit confusing and I will update the file names to be a more vague umbrella, but I think the idea is to have everything specific for this API inside that bin/api.ts file that supports deploying and hosting that service. What are your thoughts

Reply:
This also uses a new logical "App()" instead of the existing one.

Reply:
I'm not running api.ts at all when deploying the pipeline stack